hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -160,9 +162,11 @@ public class HiveSyncConfig extends HoodieSyncConfig {
 
   public static final ConfigProperty<String> HIVE_SUPPORT_TIMESTAMP_TYPE = ConfigProperty
       .key("hoodie.datasource.hive_sync.support_timestamp")
-      .defaultValue("false")
-      .withDocumentation("‘INT64’ with original type TIMESTAMP_MICROS is converted to hive ‘timestamp’ type. "
-          + "Disabled by default for backward compatibility.");
+      .defaultValue("true")
```

Comment:
please revert this default value change. it could break behaviors for existing users.